src/groups/symplectic_group.jl

```diff
@@ -0,0 +1,37 @@
+"""
+    SymplecticGroup{T}
+
+On the set of [real symplectic matrices](@extref `Manifolds.SymplecticMatrices`),
+of size ``2n×2n`` for some ``n∈ℕ``
+
+```math
+$(_math(:Sp))(2n, ℝ) = $(_tex(:SetDef, "p ∈ ℝ^{2n×2n}", "p^$(_tex(:transp))J_{2n}p = J_{2n}", "big"))
+```
+
+where ``J_{2n} = $(_tex(:pmatrix, "0_n & I_n", "-I_n & 0_n"))`` denotes the [`SymplecticElement`](@extref `Manifolds.SymplecticElement`).
```

Comment:
Maybe you could add that the group operation is matrix multiplication somewhere?

Reply:
Good idea, will do.